schedule/yast/ext4/ext4@yast.yaml

```diff
@@ -1,10 +1,9 @@
 ---
-name:           ext4@yast
```

Comment:
all ext4 scenarios should share the same part. table to be consistent, is intended the change in this file?